packages/storage-plus/src/map.rs

```diff
@@ -48,6 +48,11 @@ where
         Prefix::new(self.namespace, &p.prefix())
     }
 
+    #[cfg(feature = "iterator")]
+    fn no_prefix(&self, p: K::NoPrefix) -> Prefix<T> {
+        Prefix::new(self.namespace, &p.prefix())
```

Comment:
`p` will always be `()`. `p.prefix()` will always be `vec![]`

Why not return `Prefix::new(self.namespace, &[])` and remove the argument?